libs/perun/dialogs/src/lib/password-reset-dialog/password-reset-dialog.component.html

```diff
@@ -0,0 +1,45 @@
+<ng-template #spinner>
+  <perun-web-apps-loading-dialog></perun-web-apps-loading-dialog>
+</ng-template>
+<div class="user-theme position-relative">
+  <h1 mat-dialog-title>
+    {{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.TITLE' | translate}}
+  </h1>
+  <div class="dialog-container" mat-dialog-content>
+    <p *ngIf="data.login" class="subtitle">
+      <em
+        >{{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.LOGIN' | translate}}:
+        {{data.login}}</em
+      >
+    </p>
+    <p *ngIf="data.namespace" class="subtitle">
+      <em
+        >{{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.NAMESPACE' | translate}}:
+        {{data.namespace}}</em
+      >
+    </p>
```

Comment:
Formated. Sometimes Prettier needs small help to format things nicely.
```suggestion
    <p *ngIf="data.login" class="subtitle">
      <em>
        {{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.LOGIN' | translate}}:
        {{data.login}}
      </em>
    </p>
    <p *ngIf="data.namespace" class="subtitle">
      <em>
        {{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.NAMESPACE' | translate}}:
        {{data.namespace}}
      </em>
    </p>
```